examples/http/b3_context_in_headers.py

```diff
@@ -0,0 +1,184 @@
+"""Demonstrates a Trace distributed across multiple machines.
+
+A SpanContext's text representation is stored in the headers of an HTTP request.
+
+Runs two threads, starts a Trace in the client and passes the SpanContext to the server.
+"""
+
+import argparse
+import errno
+import socket
+import sys
+import threading
+
+try:
```

Comment:
Thanks for the example. For the next iterations we will have most likely to update it, as we need to comply with a few proposed changes about how to allow the user to specify this ;)

Reply:
Sure, it is basically the same as the previous HTTP example. I can refactor it completely or create one new from scratch too.

Reply:
The example can be executed now as it is. Please let me know when we need to update it.